src/main/java/step3/Car.java

```diff
@@ -0,0 +1,40 @@
+package step3;
+
+import java.util.Random;
+
+public class Car {
+    public static final int MIN_MOVEMENT_THRESHOLD = 4;
+    public static final String GO = "-";
+    public static final String STOP = "";
+
+    private final Random random = new Random();
+
+    private String position = "";
+
+    public void move() {
+        updatePosition(getMovementStatus(generateRandomNumber()));
+    }
```

Comment:
```suggestion
    public void move() {
        var movementStatus = getMovementStatus(generateRandomNumber())
        if (movementStatus.equals(GO)) {
            position += movementStatus;
        }
    }
```
이미 `updatePosition`자체가 단순히 메서드로 래핑한 것이기 때문에 따로 메서드로 분리할 필요없이 바로 표현해도 움직임에 대한 의미가 전달될 것 같아요 :)